ecl/eclcc/eclcc.cpp

```diff
@@ -2862,12 +2863,18 @@ int EclCC::parseCommandLineOptions(int argc, const char* argv[])
         {
             const char * arg = tempArg;
             const char * at = strchr(arg, '@');
+            const char * hash = strchr(arg, '#');
             if (at)
             {
                 optQueryMainAttribute.set(arg, at - arg);
                 optQueryMainPackage.set(at+1);
                 optExplicitMainPackage = true;
             }
+            else if (hash)
```

Comment:
What should happen if they provide both @ and #?

Reply:
A # can already follow an @ e.g., x@ghalliday/ecl-demo-d#version3